packages/core/contracts/financial-templates/common/FeePayer.sol

```diff
@@ -104,7 +104,7 @@ abstract contract FeePayer is AdministrateeInterface, Testable, Lockable {
         uint256 time = getCurrentTime();
         FixedPoint.Unsigned memory collateralPool = _pfc();
 
-        // Fetch the regualar fees, late penatly and the max posible to pay given the current collateral within the contract.
```

Comment:
regualar -> regular
posible -> possible

Reply:
Thanks for catching these. I did a search for any similar typos through the repo on these specific mistakes & we good.